app/src/sharedTest/java/org/oppia/android/app/devoptions/platformparameters/PlatformParametersFragmentTest.kt

```diff
@@ -0,0 +1,732 @@
+        )
+        verifyPlatformParameterState(
+          position = index,
+          expectedValue = ephemeralPlatformParameter.currentValue
+        )
+      }
+    }
+  }
+
+  @Test
+  fun testPlatformParametersFragment_withNoRemoteOrOverriddenFirstParameter_hasCorrectName() {
+    setUpTestApplicationComponent()
+    launch(PlatformParametersTestActivity::class.java).use {
+      testCoroutineDispatchers.runCurrent()
+
+      scrollToPosition(0)
+      verifyPlatformParameterDisplayName(
+        position = 0,
+        expectedDisplayName = SPLASH_SCREEN_WELCOME_MSG_PARAMETER_NAME
+      )
+    }
+  }
+
+  @Test
+  fun testPlatformParametersFragment_withNoRemoteOrOverriddenFirstParameter_hasCorrectSyncStatus() {
+    setUpTestApplicationComponent()
+    launch(PlatformParametersTestActivity::class.java).use {
+      testCoroutineDispatchers.runCurrent()
+
+      scrollToPosition(0)
+      verifyPlatformParameterSyncStatus(
+        position = 0,
+        expectedSyncStatus = context.getString(R.string.platform_parameter_default_sync_status)
+      )
+    }
+  }
+
+  @Test
+  fun testPlatformParametersFragment_withNoRemoteOrOverriddenFirstParameter_hasCorrectValue() {
+    setUpTestApplicationComponent()
+    launch(PlatformParametersTestActivity::class.java).use {
+      testCoroutineDispatchers.runCurrent()
+      val splashScreenWelcomeMsgParameter = getEphemeralPlatformParameters()[0]
+
+      scrollToPosition(0)
+      verifyPlatformParameterState(
+        position = 0,
+        expectedValue = splashScreenWelcomeMsgParameter.currentValue
+      )
+    }
+  }
+
+  @Test
+  fun testPlatformParametersFragment_withNoRemoteOrOverriddenFirstParameter_hasCorrectBackground() {
+    setUpTestApplicationComponent()
+    launch(PlatformParametersTestActivity::class.java).use {
+      testCoroutineDispatchers.runCurrent()
+
+      scrollToPosition(0)
+      verifyPlatformParameterBackgroundColor(
+        position = 0,
+        expectedColor = 0xFFBE563C.toInt()
+      )
+    }
+  }
+
+  @Test
+  fun testPlatformParametersFragment_splashScreenWelcomeMsgParameterSwitchToggled_updatesValue() {
+    setUpTestApplicationComponent()
+    launch(PlatformParametersTestActivity::class.java).use {
+      testCoroutineDispatchers.runCurrent()
+      val splashScreenWelcomeMsgParameter = getEphemeralPlatformParameters()[0]
+
+      scrollToPosition(0)
+      onView(
+        atPositionOnView(
+          recyclerViewId = R.id.platform_parameters_recycler_view,
+          position = 0,
+          targetViewId = R.id.platform_parameter_switch
+        )
+      ).perform(click())
+
+      verifyPlatformParameterState(
+        position = 0,
+        expectedValue = PlatformParameterValue.newBuilder()
+          .setBoolean(!splashScreenWelcomeMsgParameter.currentValue.boolean)
+          .build()
+      )
+    }
+  }
+
+  @Test
+  fun testPlatformParametersFragment_toggleSplashScreenWelcomeMsg_configChanges_valuePersists() {
```

Comment:
Aligning with PR 1.2 [FeatureFlagsFragmentTest]
```suggestion
  fun testPlatformParametersFragment_toggleSplashScreenWelcomeMsg_configChange_persistsValue() {
```

Reply:
updated

Reply:
Double-checking is this updated -> `configChange_persistsValue()`?

Reply:
updated now

Reply:
Can you recheck? `configChange` or did I miss anything?

>Review request based from PR 1.1: https://github.com/oppia/oppia-android/pull/5861/files/41c5c0670b363f66b580334b23c00a20dd5bdb2c#r2183710029

Reply:
updated now